### PR TITLE
U19CS066

### PR DESCRIPTION
**Pull request by mahitha0x01**

### CODE REVIEW

File: stack.c [hunk 1]
```diff
@@ -5,55 +5,47 @@
...
+    if(stack->top == -1) {
         printf("Stack underflow while pop\n");
         return -1;
     }
-    Node* next = stack->top->next;
-    stackType data = stack->top->data;
-    free(stack->top);
-    stack->top = next;
-    return data;
+    stack->top == -1;
```
Comment (murtraja) [inline stack.c]: Did you run your code before submitting?

File: stack.c [hunk 2]
```diff
@@ -5,55 +5,47 @@
 #include "stack.h"
 
 void push(Stack* stack, stackType data) {
-    Node* newNode = malloc(sizeof(Node));
-    if(newNode == NULL) {
+    if(stack->top == MAX_SIZE) {
```
Comment (murtraja) [inline stack.c]: MAX_SIZE -1?

File: stack.c (hunk 1 above)
Comment (mahitha0x01) [reply stack.c]: yes sir it didnt give the proper output 
I didnt know the mistake

File: stack.c (hunk 2 above)
Comment (mahitha0x01) [reply stack.c]: ok I'll make the changes again sir

**Comment by murtraja** [comment]
Looks good.

**Pull request closed without merging**